guides/common/modules/proc_deploying-a-policy-in-a-host-group-using-puppet.adoc

```diff
@@ -0,0 +1,24 @@
+[id="Deploying_a_Policy_in_a_Host_Group_Using_Puppet_{context}"]
+= Deploying a Policy in a Host Group Using Puppet
+
+Use this procedure to configure all the OpenSCAP requirements for a host group.
+
+.Prerequisites
+* You have enabled OpenSCAP on {SmartProxy}.
+ifdef::satellite[]
+For more information, see {InstallingSmartProxyDocURL}Enabling_OpenSCAP_on_External_Smart_Proxies_{smart-proxy-context}[Enabling OpenSCAP on External {SmartProxies}] in _{InstallingSmartProxyDocTitle}_.
+endif::[]
+ifndef::satellite[]
+For more information, see xref:Installing_the_OpenSCAP_Plugin_{context}[].
+endif::[]
+* You have enabled and synced the {project-client-name} repository to {Project} and enabled it on the hosts.
+* You have xref:Creating_a_Compliance_Policy_{context}[created a compliance policy] with the Puppet deployment option and assigned the host group.
+
+.Procedure
+. In the {ProjectWebUI}, navigate to *Configure* > *Host Groups*.
+. Click the host group that you want to configure for OpenSCAP reporting.
+. In the *Environment* list, select the Puppet environment that contains the `foreman_scap_client*` Puppet classes.
+. In the *OpenSCAP {SmartProxy}* list, select the {SmartProxy} with OpenSCAP enabled that you want to use.
+. On the *Puppet ENC* tab, add the `foreman_scap_client` and `foreman_scap_client::params` Puppet classes.
+. Optional: Configure any *Puppet Class Parameters*.
+. Click *Submit* to save your changes.
```

Comment:
Is this missing the last sentence from the procedure for Ansible? Or an explanation that Puppet runs automatically within the next 30 minutes?